logstash-filter-grok.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'stud', '~> 0.0.22'
   s.add_runtime_dependency 'logstash-patterns-core'
 
-  s.add_development_dependency 'logstash-devutils'
+  s.add_development_dependency 'logstash-devutils', '= 1.3.4'
```

Comment:
this should be necessary any more (1.3.6 has been released that fixes the 5.x compatibility)

Reply:
@jakelandis hmm it wasn't pulling that version in for me locally, probably best to fix at `1.3.6` then right?

Reply:
It doesn't hurt either way. Odd that it wasn't pulling the latest (it was just released however).